airbyte-integrations/connectors/source-alpha-vantage/unit_tests/test_object_dpath_extractor.py

```diff
@@ -0,0 +1,169 @@
+import json
+from typing import Any
+
+from requests import Response
+
+from source_alpha_vantage.object_dpath_extractor import ObjectDpathExtractor
```

Comment:
Nice! thanks for the implementation of the unit test.

Reply:
No worries. It would be great to see something like this implemented in the base `DpathExtractor`. Perhaps as a config option to avoid breaking changes?